packages/calcite-components/src/components/shell-panel/shell-panel.scss

```diff
@@ -288,15 +288,27 @@ slot[name="action-bar"]::slotted(calcite-action-bar),
   @apply border-color-3 border border-solid;
 }
 
-:host([position="start"]) slot[name="action-bar"]::slotted(calcite-action-bar),
-:host([position="start"]) .content ::slotted(calcite-flow),
-:host([position="start"]) .content ::slotted(calcite-panel) {
+:host([position="start"]:not([slot="panel-end"])) slot[name="action-bar"]::slotted(calcite-action-bar),
```

Comment:
Can you use nesting to reduce some duplication?

Applies to other updated, grouped selectors.

Reply:
Initially had nesting and it affected the readability to some extent. Thoughts?

Reply:
Ex:
``` 
:host([position="end"][slot="panel-start"]),
:host([position="end"][slot="panel-start"]) {
  slot[name="action-bar"]::slotted(calcite-action-bar),
  .content ::slotted(calcite-flow),
  .content ::slotted(calcite-panel) {
    border-inline-start: none;
  }
}